python/tvm/contrib/hexagon/build.py

```diff
@@ -217,10 +233,7 @@ def load_module(self, module: Union[str, pathlib.Path, tvm.runtime.Module], sess
             session and loaded.
 
             If the object passed is a string or pathlib.Path, it must
-            be either a bare file name (without any path components),
-            or a full path in the remote system. If it is a file name,
-            the file must already have been uploaded to the remote,
-            and be placed in the remote workspace.
+            be a full path in the remote system.
```

Comment:
I like the symmetry of loading a file using the same path as returned from `upload`, but are there cases where we'd want to work within the existing workspace, without requiring the full path?  The use cases coming to mind would be uploading a model once, then running several independent tests on it.  Within a single process, the model to run should be passed to each independent target, so this change wouldn't affect it.  If the upload happens in one process, then the independent tests happen in subsequent processes, it might cause an issue, but then the workspace itself would need to be passed through.

So I don't think I see any major concerns, but reducing previously functional behavior does give me pause.